src/RealtimeClient.ts

```diff
@@ -111,7 +113,9 @@ export default class RealtimeClient {
    */
   constructor(endPoint: string, options?: RealtimeClientOptions) {
     this.endPoint = `${endPoint}/${TRANSPORTS.websocket}`
-
+    this.httpEndpoint = endPoint
```

Comment:
@filipecabaco there's already similar logic here: https://github.com/supabase/realtime-js/blob/2798441cbfa6f6dc665b4adcafbfd99d5f8a40f3/src/RealtimeChannel.ts#L531. We can move to a helper file and use in both places.

Reply:
awesome! ok will tackle that refactor 👍 thank you for the heads up